tailwind.base.config.js

```diff
@@ -19,17 +19,18 @@ module.exports = {
         'secondary-darker': 'var(--color-secondary-darker)',
         'secondary-darkest': 'var(--color-secondary-darkest)',
         main: 'var(--color-main)',
+        title: 'black',
         background: 'var(--color-background)',
-        gray: {
-          100: 'var(--color-gray-100)',
-          200: 'var(--color-gray-200)',
-          300: 'var(--color-gray-300)',
-          400: 'var(--color-gray-400)',
-          500: 'var(--color-gray-500)',
-          600: 'var(--color-gray-600)',
-          700: 'var(--color-gray-700)',
-          800: 'var(--color-gray-800)',
-          900: 'var(--color-gray-900)',
```

Comment:
So as discussed I'm not really in favor of this change, I think it's confusing to have both a `text-main` class and a `text-main-xxx` scale; and I think we can safely rely on `gray-xxx` being actually gray since main is supposed to be close to black (or white in a dark theme). But overall it's not breaking anything so I'm leaving this up to you :slightly_smiling_face:

Reply:
Hum, I'm wondering if we actually still need these scaled classes, since tailwind provides the `gray` scales and the renamed `main` scales are currently not used. Regarding the colors, neither the tailwind grays nor the custom ones match the mockup's grays RGB a 100%, but we get very close with both. After all, it's a decision if we want the gray scale dependence to our main color or not. If it's not essential, I propose to remove these classes and their calculation for the sake of simplicity.

Reply:
After re-discussing with @fgravin I finally, kept it as it was before.